cmd/oras/root/repo/tags.go

```diff
@@ -115,10 +131,17 @@ func showTags(cmd *cobra.Command, opts *showTagsOptions) error {
 	}
 	err = finder.Tags(ctx, opts.last, func(tags []string) error {
 		for _, tag := range tags {
+			if targetPrefix != "" {
```

Comment:
Can we add a few comments for each if statement like this for better maintainability?